server/src/main/java/io/spine/server/procman/ProcessManagerRepository.java

```diff
@@ -100,36 +103,70 @@ protected ProcessManagerRepository() {
      *
      * <p>Registers with the {@code CommandBus} for dispatching commands
      * (via {@linkplain DelegatingCommandDispatcher delegating dispatcher}).
+     *
+     * <p>Registers with the {@code IntegrationBus} for dispatching external events and rejections.
+     *
+     * <p>Ensures there is at least one message handler declared by the class of the managed
+     * process manager:
+     *
+     * <ul>
+     *     <li>command handler methods;</li>
+     *     <li>domestic or external event reactor methods;</li>
+     *     <li>domestic or external rejection reactor methods.</li>
+     * </ul>
+     *
+     * <p>Throws an {@code IllegalStateException} otherwise.
      */
+    @SuppressWarnings("MethodWithMoreThanThreeNegations")   // It's fine, as reflects the logic.
     @Override
     public void onRegistered() {
         super.onRegistered();
-        final BoundedContext boundedContext = getBoundedContext();
 
-        final DelegatingCommandDispatcher<I> commandDispatcher =
-                DelegatingCommandDispatcher.of(this);
+        final BoundedContext boundedContext = getBoundedContext();
+        final DelegatingRejectionDispatcher<I> rejDispatcher =
+                DelegatingRejectionDispatcher.of(this);
 
-        if (!commandDispatcher.getMessageClasses()
-                              .isEmpty()) {
-            boundedContext.getCommandBus()
-                          .register(commandDispatcher);
+        final boolean handlesCommands = register(boundedContext.getCommandBus(),
+                                                 DelegatingCommandDispatcher.of(this));
+        final boolean handlesDomesticRejections = register(boundedContext.getRejectionBus(),
+                                                           rejDispatcher);
+        final boolean handlesExternalRejections = register(boundedContext.getIntegrationBus(),
+                                                           rejDispatcher.getExternalDispatcher());
+        final boolean handlesDomesticEvents = !getMessageClasses().isEmpty();
+        final boolean handlesExternalEvents = !getExternalEventDispatcher().getMessageClasses()
+                                                                          .isEmpty();
+
+        final boolean subscribesToEvents = handlesDomesticEvents || handlesExternalEvents;
+        final boolean reactsUponRejections = handlesDomesticRejections || handlesExternalRejections;
+
+        if (!handlesCommands && !subscribesToEvents && !reactsUponRejections) {
+            throw newIllegalStateException(
```

Comment:
This branch appears not covered.

Reply:
This might be a bug on __codecov__, but please check for sure.

Reply:
Double-checked. 

Yes, that looks like an outdated **codecov** report.